packages/cli/src/commands/init/template.js

```diff
@@ -44,5 +49,5 @@ export function executePostInitScript(
 
   logger.debug(`Executing post init script located ${scriptPath}`);
 
-  execFileSync(scriptPath, {stdio: 'inherit'});
+  return promisify(execFile)(scriptPath, {stdio: 'inherit'});
```

Comment:
use `execa` instead of `child_process`, it accepts file as a first parameter and has promise interface

Reply:
I was thinking the same but didn't want to introduce the next dependency. I'll gladly use `execa` because testing will be much easier.

Reply:
fortunately we already use it!

Reply:
OH, SICK!